src/Http/Response/JSON/Accepted.php

```diff
@@ -0,0 +1,40 @@
+<?php declare(strict_types=1);
```

Comment:
Surprised this doesn't exist but if you move it to a different PR it could be easily merged.